src/test/java/com/emc/mongoose/storage/driver/pravega/integration/PravegaEventReadTest.java

```diff
@@ -56,7 +58,8 @@ public void testEventRead()
 			readerGroupManager.createReaderGroup(readerGroup, readerGroupConfig);
 		}
 		try (
-						val clientFactory = ClientFactory.withScope(scopeName, controllerURI);
+				val clientFactory = EventStreamClientFactory.withScope(scopeName,
+						ClientConfig.builder().controllerURI(controllerURI).build());
```

Comment:
Same for the tab intent. Shouldn't it be like that: https://github.com/emc-mongoose/mongoose-storage-driver-pravega/blob/master/src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java#L532 ?